src/parser/shadowlands/modules/Mikanikos.ts

```diff
@@ -0,0 +1,92 @@
+import SPELLS from 'common/SPELLS';
+import { CLASSES, getClassBySpecId } from 'game/CLASSES';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import SOULBINDS from 'game/shadowlands/SOULBINDS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { CastEvent, ApplyDebuffEvent } from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+
+interface SpellInfo {
+  id: number;
+  name: string;
+}
+
+const SPELLS_BY_CLASS: { [key: number]: SpellInfo } = {
+  [CLASSES.DEATH_KNIGHT]: SPELLS.SHACKLE_THE_UNWORTHY,
+  [CLASSES.DEMON_HUNTER]: SPELLS.ELYSIAN_DECREE,
+  [CLASSES.DRUID]: SPELLS.KINDRED_SPIRITS,
+  [CLASSES.HUNTER]: SPELLS.RESONATING_ARROW,
+  [CLASSES.MAGE]: SPELLS.RADIANT_SPARK,
+  [CLASSES.MONK]: SPELLS.WEAPONS_OF_ORDER,
+  [CLASSES.PALADIN]: SPELLS.DIVINE_TOLL,
+  [CLASSES.PRIEST]: SPELLS.BOON_OF_THE_ASCENDED,
+  [CLASSES.ROGUE]: SPELLS.ECHOING_REPRIMAND,
+  [CLASSES.SHAMAN]: SPELLS.VESPER_TOTEM,
+  [CLASSES.WARLOCK]: SPELLS.SCOURING_TITHE,
+  [CLASSES.WARRIOR]: SPELLS.SPEAR_OF_BASTION,
+};
+
+class Mikanikos extends Analyzer {
```

Comment:
This should probably be called EffusiveAnimaAccelerator as that is what the module is for. Then it would fit under shadowlands/covenants/kyrian or something similarly named